releasinator.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
`mkvirtualenv cardio` will make the virtual environment called "cardio"  , remaining commands need to be executed under cardio virtual environment,  not at regular command prompt location.   if  I use  `CommandProcessor.command` it's executing commands at regular command prompt location